assets/js/base/components/form-token-field/vendor.scss

```diff
@@ -4,9 +4,9 @@
 // Tooltip must be under the same scope to work properly.
 .wc-blocks-components-form-token-field-wrapper {
 	/* stylelint-disable no-invalid-position-at-import-rule */
-	@import "@wordpress/base-styles/breakpoints";
-	@import "@wordpress/base-styles/mixins";
-	@import "wordpress-components/src/popover/style";
-	@import "wordpress-components/src/tooltip/style";
+	@import "node_modules/@wordpress/base-styles/breakpoints";
```

Comment:
I should add a comment with a link to this PR. In this way, in the future, it will be easier to discover why it is necessary to import the CSS in this way.